.github/workflows/version.yml

```diff
@@ -126,3 +118,26 @@ jobs:
           title: '[AUTO] [main] Add bwc version ${{ env.NEXT_VERSION }}.'
           body: |
             I've noticed that a new tag ${{ env.TAG }} was pushed, and added a bwc version ${{ env.NEXT_VERSION }}.
+
+      - name: Create tracking issue
```

Comment:
I agree. This is just a way to make the changes traceable and will not solve the build failures.
That said, I am a bit hesitant to push directly to the branches - which will also lead to the same failures if your forked branch is not synchronized.

Given this is not a frequent scenario - the PR model seems like a good tradeoff.

Reply:
I've updated the PR description to say Related rather than Resovles so that issue isn't closed when this change is merged. 

![image](https://github.com/opensearch-project/OpenSearch/assets/2754967/93385aa8-7651-4692-bc5f-9d822fc0ee0c)